docs/en/reference/advanced-configuration.rst

```diff
@@ -317,7 +317,7 @@ Proxy Objects
 
 A proxy object is an object that is put in place or used instead of
 the "real" object. A proxy object can add behavior to the object
-being proxied without that object being aware of it. In Doctrine 2,
+being proxied without that object being aware of it. In ORM,
```

Comment:
```suggestion
being proxied without that object being aware of it. In Doctrine ORM,
```

Reply:
This was intentional. I wanted to reduce "Doctrine" appearing everywhere to prevent it becoming too redundant. I actually thinking about reducing its occurrence in annotations-reference.rst where it is used multiple times.